src/Scene.cpp

```diff
@@ -0,0 +1,130 @@
+#include <glad/glad.h>
+
+#include "Scene.h"
+#include "ShaderTool.h"
+
+Scene::Scene(std::string vertex_shader, std::string fragment_shader)
```

Comment:
Do we want scene to only have 1 program, or a list of program?

Reply:
I think we can keep it to 1 program for now. We can add more shaders later when necessary.